handlers/webhook.js

```diff
@@ -17,6 +17,16 @@ const challenge = function *(request, slackMessage, h) {
 
     const challengingPlayer = yield KingPong.getPlayerFromId(slackMessage.user_id);
 
+    if (challengedPlayerId.toLowerCase == "winner") {
```

Comment:
Deze variabele bestaat hier nog niet, die wordt hieronder pas gemaakt

Reply:
Daarnaast is het volgens mij ook gewoon een int dus vergelijken met 'winner' gaat sowieso fout toch?